scenarios/python/console/captioning/README.md

```diff
@@ -0,0 +1,55 @@
+# Scenarios: Create captions with speech to text
+In this quickstart, you run a console app to create captions with speech to text.
+
+## Run the Sample within VS Code
+1. Install "Azure AI Speech Toolkit" extension in VS Code.
+2. Download this sample from sample gallery to local machine.
+3. Trigger "Azure AI Speech Toolkit: Configure Azure Speech Resources" command from command palette to select speech resource.
+4. Trigger "Azure AI Speech Toolkit: Build the Sample App" command from command palette to build the sample.
+5. Trigger "Azure AI Speech Toolkit: Run the Sample App" command from command palette to run the sample.
+6. You can choose to use a specified input file or use the microphone as input data by default.
+7. In other IDEs you can refer to [captioning-quickstart](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/captioning-quickstart?tabs=windows%2Cterminal&pivots=programming-language-python)
```

Comment:
## Run the Sample within VS Code
1. Install "Azure AI Speech Toolkit" extension in VS Code.
2. Download this sample from sample gallery to local machine.
3. Trigger "Azure AI Speech Toolkit: Configure Azure Speech Resources" command from command palette to select speech resource.
4. Trigger "Azure AI Speech Toolkit: Build the Sample App" command from command palette to build the sample.
5. Trigger "Azure AI Speech Toolkit: Run the Sample App" command from command palette to run the sample. This command will run the sample  as a task in terminal. Interactively input local audio file if you would like to use file instead of default microphone as audio input.

## Run the Sample out of VS Code
In other IDEs, please refer to [captioning-quickstart](https://learn.microsoft.com/en-us/azure/ai-services/speech-service/captioning-quickstart?tabs=windows%2Cterminal&pivots=programming-language-python) for a detailed guide.